bigquery_etl/util/udf_functions.py

```diff
@@ -0,0 +1,29 @@
+"""Utility functions related to UDF parsing and documentation."""
+
+import re
+
+
+def find_input(sql_text):
+    """Extract input parameters from UDF sql code."""
+    sql_text = re.sub(r"\s{2,}|\\n", " ", sql_text)
+    return (
+        re.search(r"FUNCTION [a-z,0-9,_,.]+\s*\(\s*(.+?)\s*\)", sql_text)
```

Comment:
Just want to check that function names are guaranteed to be lowercase?

I also think we can narrow the whitespace searching a little bit because we already removed all whitespace of 2+ characters in the line 8 (though I'm not actually sure if this is more efficient performance-wise?):
```suggestion
        re.search(r"FUNCTION [a-z,0-9,_,.]+\s?\(\s?(.+?)\s?\)", sql_text)
```

Reply:
They do have comments but I'm looking for a specific string pattern around the group I'm looking for. In the tests it worked with lot's of comments ... we would only run into problems here if someone would comment out an old udf and add a new one underneath I think

Reply:
But if you have a specific problem in mind, we could add that to the tests and see if it works

Reply:
on the function names ... I'm not 100% sure that we are guaranteed it's lowercase but I think it's the convention .. nut sure if it's better to allow for everything or keep in line with the convention here, open for suggestions